cob_bringup/robots/raw3-6.xml

```diff
@@ -44,23 +45,29 @@
 			<arg name="robot" value="$(arg robot)" />
 		</include>
 
-                <include file="$(find cob_bringup)/drivers/phidgets.launch" >
-                        <arg name="robot" value="$(arg robot)" />
-                </include>
+		<include file="$(find cob_bringup)/drivers/phidgets.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
 
-                <include file="$(find cob_bringup)/drivers/powerstate_phidget.launch">
-                        <arg name="robot" value="$(arg robot)"/>
-                </include>
+		<include file="$(find cob_bringup)/drivers/powerstate_phidget.launch">
+			<arg name="robot" value="$(arg robot)"/>
+		</include>
 
-                <include file="$(find cob_bringup)/drivers/emstate_phidget.launch">
-                        <arg name="robot" value="$(arg robot)"/>
-                </include>
+		<include file="$(find cob_bringup)/drivers/emstate_phidget.launch">
+			<arg name="robot" value="$(arg robot)"/>
+		</include>
 
-                <include file="$(find cob_bringup)/tools/battery_monitor.launch" >
-                        <arg name="robot" value="$(arg robot)" />
-                </include>
+		<include file="$(find cob_bringup)/tools/battery_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
 
 		<!-- upper robot -->
+		<include file="$(find ur_bringup)/launch/ur_common.launch">
```

Comment:
This launch file includes also a robot_state_publisher node https://github.com/ros-industrial/universal_robot/blob/indigo-devel/ur_bringup/launch/ur_common.launch#L28 

`roslaunch file contains multiple nodes named [/robot_state_publisher]`